sources/engine/Stride.VirtualReality/OpenXR/OpenXRHmd.cs

```diff
@@ -19,7 +20,7 @@ public class OpenXRHmd : VRDevice
         // Public static variable to add extensions to the initialization of the openXR session
         public static List<string> extensions = new List<string>();
 
-        public static OpenXRHmd? New()
+        public static OpenXRHmd? New(bool requestPassthrough)
```

Comment:
Since this is public API can you add an XML comment to this method explaining the parameter?